python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -138,6 +139,11 @@ class Node extends TNode {
    * Gets a local source node from which data may flow to this node in zero or more local steps.
    */
   LocalSourceNode getALocalSource() { result.flowsTo(this) }
+
+  /**
+   * Gets a local source node from which data may flow to this node in zero or more local steps.
```

Comment:
```suggestion
   * Gets a local source node from which data may flow to this node in zero or more local taint-flow steps.
```
... And I would likewise add "data-flow" to `getALocalSource", to make it clear what the difference between these methods is.